### ISSUE TITLE
Dynamic linking error: cannot resolve symbol emscripten_compute_dom_pk_code

### ISSUE THREAD

**Issue by maartenbreddels** [pyodide/pyodide#5029]
## 🐛 Bug

```
pyodide.ffi.JsException: Error: Dynamic linking error: cannot resolve symbol emscripten_compute_dom_pk_code
```

### To Reproduce

https://py.cafe/maartenbreddels/solara-pygame-breakpong

Click on the canvas, and hit enter.

I've added a comment 'CRASH' where the exception gets triggered.

### Expected behavior

To have emscripten/pygame-ce/sdl process the keyboard event, like it does with mouse events (I confirmed that)

### Environment

- Pyodide Version: 0.26.1 (can be changed in settings)
- Browser version:  Version 1.68.137 Chromium: 127.0.6533.100

**Comment by hoodmane** [comment]
Well it looks like `emscripten_compute_dom_pk_code` is included in `libhtml5.a` which we don't link (as opposed to `libhtml5.js` which we do link with `-lhtml5.js`). If I do pass `-lhtml5` to the linker, I get an error because `emscripten_wget_data` is undefined. But `emscripten_wget_data` requires `-sASYNCIFY`. If we add an empty defintion:
```C
void emscripten_wget_data(const char* url, void** pbuffer, int* pnum, int *perror) {}
```
then everything works as expected. Or if we apply this patch:
https://github.com/emscripten-core/emscripten/pull/22435

**Comment by maartenbreddels** [comment]
Thanks for looking into this 🙏

**Comment by ryanking13** [comment]
This should be fixed in pyodide 0.26.2 (https://github.com/pyodide/pyodide/pull/4865).

**Comment by hoodmane** [comment]
I'm pretty sure it's not fixed by #4865 because that added `-lhtml5.js` but this requires also passing `-lhtml5`. You can test it out directly on dev and you see that the symbol is undefined:
```js
> pyodide._module.resolveGlobalSymbol("emscripten_compute_dom_pk_code")
< {sym: undefined, name: 'emscripten_compute_dom_pk_code'}
```

**Comment by ryanking13** [comment]
Yes, `emscripten_compute_dom_pk_code` is linked only to the pygame wheel (.so), not in pyodide.asm.js, and it is intentional.

When linking shared libraries, I try to link only the JS part to the main module, while linking native codes to the wheel, to reduce the size of the main module. For example, when it comes to SDL libraries, we only link the JS part to pyodide.asm.js, the rest is linked directly to the corresponding pygame or pyxel wheel.

**Comment by maartenbreddels** [comment]
It was indeed fixed by 0.26.2, I just added support in pycafe for that, and this game seems to run now:
 https://py.cafe/maartenbreddels/solara-pygame-breakpong

Thanks, and sorry for not checking the latest version.

Is the alpha version released on npm?

**Comment by hoodmane** [comment]
Yeah you can get the most recent release including unstable ones with `npm i pyodide@next`

**Comment by maartenbreddels** [comment]
Going a but off topic here:

But I don't think I can load that from pycafe.
i.e.

https://cdn.jsdelivr.net/pyodide/v0.26.0/full/pyodide.js loads 
https://unpkg.com/pyodide@0.27.0-alpha.2/pyodide.js load (no unpkg cdn supports alphas?)
But 
https://cdn.jsdelivr.net/pyodide/v0.27.0-alpha.2/full/pyodide.js does not

So it seems jsdelivr does not support alphas?

**Comment by hoodmane** [comment]
It's just spelled differently: v0.27.0a2 instead of v0.27.0-alpha.2

**Comment by agriyakhetarpal** [comment]
This issue was resolved with Pyodide 0.26.2 and can be closed now. Thanks!
